### PR TITLE
Shorten test_output folder name to output

### PR DESCRIPTION
**Pull request by renfeiw**
Signed-off-by: renfeiw <renfeiw@ca.ibm.com>

### CODE REVIEW

**Comment by renfeiw** [comment]
related PR: https://github.com/AdoptOpenJDK/TKG/pull/132

File: buildenv/jenkins/JenkinsfileBase [hunk 1]
```diff
@@ -626,28 +626,28 @@ def post(output_name) {
 			if ((currentBuild.result == 'UNSTABLE' || currentBuild.result == 'FAILURE' || currentBuild.result == 'ABORTED') || params.ARCHIVE_TEST_RESULTS) {
 				def test_output_tar_name = "${output_name}_test_output${suffix}"
 				if (tar_cmd.startsWith('tar')) {
-					sh "${tar_cmd} - ${pax_opt} ./openjdk-tests/TKG/test_output_* ${tar_cmd_suffix} > ${test_output_tar_name}"
+					sh "${tar_cmd} - ${pax_opt} ./openjdk-tests/TKG/output_* ${tar_cmd_suffix} > ${test_output_tar_name}"
 				} else {
-					sh "${tar_cmd} ${test_output_tar_name} ${pax_opt} ./openjdk-tests/TKG/test_output_* ${tar_cmd_suffix}"
+					sh "${tar_cmd} ${output_tar_name} ${pax_opt} ./openjdk-tests/TKG/output_* ${tar_cmd_suffix}"
```
Comment (llxia) [inline buildenv/jenkins/JenkinsfileBase:631]: `test_output_tar_name ` is variable name. Unless we update the variable name, this cannot be changed.

Comment (renfeiw) [reply buildenv/jenkins/JenkinsfileBase]: updated

**Review by llxia — APPROVED** [review]
LGTM

**Review by smlambert — APPROVED** [review]

**Pull request merged**